specs/binding-proxy.spec/src/main/resources/META-INF/zilla/proxy.idl

```diff
@@ -127,4 +127,38 @@ scope proxy
             ProxyInfo[] infos;
         }
     }
+
+    scope event
+    {
+        enum ProxyEventType (uint8)
+        {
+            REMOTE_ACCESS_FAILED (1),
+            TLS_FAILED (2)
+        }
+
+        enum TlsError (uint8)
+        {
+            PROTOCOL_ERROR (1),
+            KEY_ERROR (2),
+            HANDSHAKE_ERROR (3),
+            PEER_UNVERIFIED_ERROR (4),
+            UNSPECIFIED_ERROR (5)
+        }
```

Comment:
Since these are `TLS` specific errors, they probably belong in `events-tls.spec` instead, a bit like metrics except that is implementation.

We might end up moving the others to `events-http.spec`, etc as well.

Reply:
I moved all `tls` specific events to a file `binding-tls.spec/src/main/resources/META-INF/zilla/tls.idl`. This is consistent with how it is for other bindings. I hope it's OK. 

The events in `proxy.idl` are now being used by the `tcp` binding only. So, maybe it'd make sense to move them to a `binding-tcp.spec/src/main/resources/META-INF/zilla/tcp.idl` file, so it's completely consistent with all the other binding events. Pls LMK if I should make this change.

Reply:
I actually went ahead with creating the `tcp.idl` file, I hope it's OK. This way there are no event related changes in the `proxy.idl` anymore and all the bindings have their event definitions in their own `<binding>.idl` files.